prestapyt/prestapyt.py

```diff
@@ -86,7 +90,8 @@ class PrestaShopWebService(object):
     """Interact with the PrestaShop WebService API, use XML for messages."""
 
     MIN_COMPATIBLE_VERSION = '1.4.0.17'
-    MAX_COMPATIBLE_VERSION = '1.7.5.2'
+    # To avoid constant version changes
+    MAX_COMPATIBLE_VERSION = '1.7.8.99'
```

Comment:
I don't think this compatibility is assured and that's why is locked.

Reply:
Pedro, my 5 cents here. This is minor version of release
It is even not 3rd version digit, but 4th. So this versions of Prestashop contains only bugfixes

Most of our customers using 1.7.8 with prestapyt library. So 1.7.8.X is supported for sure. So it doesn't make sense to validate is it 1.7.8.6 or is it 1.7.8.8 version there are only bugfixes

If you even check Prestashop addons webshop, when you purchase module it is asking which version of prestashop you are using. 

And in this list 4th digits are not taken into account. See below screenshot. So it is pretty much safe to put 99 as last digit validation. That is easy fix that do not require to rework version validation logic
![image](https://user-images.githubusercontent.com/2680904/184194018-e90e2893-a9a2-4381-b39c-47547b66ba32.jpeg)

Reply:
If you both assure that there's no compatibility problem, OK for me, but a precision: it's a change on the third digit, from 5 to 8

Reply:
Yes, 1.7.8 for sure supported. All our customers are on it (if they are not on 1.6.x )

Reply:
Are they using this library "tricked" for not having the check and there's no compatibility problem?

Reply:
OK, I didn't remember about not being blocking.